lib/govuk_index/presenters/indexable_content_presenter.rb

```diff
@@ -0,0 +1,54 @@
+module GovukIndex
+  class IndexableContentPresenter
+    DEFAULTS = %w(body parts).freeze
+    BY_FORMAT = {
+      'contact'     => %w(title description),
+      'licence'     => %w(licence_short_description licence_overview),
+      'transaction' => %w(introductory_paragraph more_information),
+    }.freeze
+
+    def initialize(format:, details:, sanitiser:)
+      @format    = format
+      @details   = details
+      @sanitiser = sanitiser
+    end
+
+    def indexable_content
```

Comment:
naming wise this is not idea `IndexableContent#indexable_content`. We should revisit how we are naming both the classes and the methods, but lets do that in the next PR. :)